### PR TITLE
Automatically bump version in pyproject.toml

### PR DESCRIPTION
**Pull request by mjste**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "scikit-fingerprints"
-version = "1.6.1"
+version = "0.0.0"   # currently poetry version is set by poetry-version-plugin from tag
```
Comment (j-adamczyk) [inline pyproject.toml:3]: Maybe reword a bit:
```
# version is read with poetry-version-plugin on GitHub release from Git tag
```

**Review by j-adamczyk — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (mjste) [reply pyproject.toml]: done
